bindata/network/openshift-sdn/002-rbac.yaml

```diff
@@ -54,6 +54,19 @@ rules:
   - patch
   - update
 
+- apiGroups:
+  - "toleration.scheduling.openshift.io"
+  resources:
+  - node-role.kubernetes.io/master
+  - node.kubernetes.io/unreachable
+  - node.kubernetes.io/not-ready
```

Comment:
So does this only add permission for those specific tolerations? Network plugins need to tolerate all possible taints (you don't want the network to go down because the node is low on disk space, etc), so the roles in `bindata/network/kuryr/001-rbac.yaml`, `bindata/network/multus/002-rbac.yaml`, `bindata/network/openshift-sdn/002-rbac.yaml`, and `bindata/network/ovn-kubernetes/002-rbac.yaml` need to reflect that.

Also, you need to add permissions (for all tolerations) to `bindata/kube-proxy/001-rbac.yaml`, which was probably added to the repo after you started working on this patch.

Reply:
Ok, let me look into that. Thanks :)

Reply:
@danwinship - Yes, this add permissions for those tolerations only. This PR was just to give everyone an idea. I'll make more changes depending on the existing tolerations for any DS or deployment.yaml.